docs/data/flink_connectors.yml

```diff
@@ -51,11 +51,11 @@ hbase:
   compatibility: ["1.16.x", "1.17.x"]
 
 jdbc:
-  name: "Apache Flink JDBC Connector 3.1.2"
-  source_release_url: "https://www.apache.org/dyn/closer.lua/flink/flink-connector-jdbc-3.1.2/flink-connector-jdbc-3.1.2-src.tgz"
-  source_release_asc_url: "https://downloads.apache.org/flink/flink-connector-jdbc-3.1.2/flink-connector-jdbc-3.1.2-src.tgz.asc"
-  source_release_sha512_url: "https://downloads.apache.org/flink/flink-connector-jdbc-3.1.2/flink-connector-jdbc-3.1.2-src.tgz.sha512"
-  compatibility: ["1.16.x", "1.17.x", "1.18.x"]
+  name: "Apache Flink JDBC Connector 3.2.0"
+  source_release_url: "https://www.apache.org/dyn/closer.lua/flink/flink-connector-jdbc-3.2.0/flink-connector-jdbc-3.2.0-src.tgz"
+  source_release_asc_url: "https://downloads.apache.org/flink/flink-connector-jdbc-3.2.0/flink-connector-jdbc-3.2.0-src.tgz.asc"
+  source_release_sha512_url: "https://downloads.apache.org/flink/flink-connector-jdbc-3.2.0/flink-connector-jdbc-3.2.0-src.tgz.sha512"
+  compatibility: ["1.18.x", "1.19.x"]
```

Comment:
[NIT]
Earlier jdbc connector used to support 3 versions (1.16.x , 1.17.x and 1.18.x) 
now it supports only 2 ( 1.18.x and 1.19.x) versions only. any specific reason ?

Reply:
The bylaws requires supporting at least two versions, 3 versions is not necessary